tests/snapshots/tests.contrib.aiohttp.test_aiohttp_client.test_trace_query_string.json

```diff
@@ -0,0 +1,37 @@
+[[
+  {
+    "name": "aiohttp.request",
+    "service": null,
+    "resource": "aiohttp.request",
+    "trace_id": 0,
+    "span_id": 1,
+    "parent_id": 0,
+    "type": "http",
+    "meta": {
+      "http.method": "GET",
+      "http.query.string": "k1=v1,k2%3Dv2&k3=v3&k4=v4",
```

Comment:
It's really strange how `=` gets encoded as `%3D` for the second param passed through the URL but that's just how the URL parsing/unparsing/stringifying seems to work 🤷